examples/vsi-with-storage/main.tf

```diff
@@ -0,0 +1,32 @@
+#####################################################
+# vsi with storage Example
+# Copyright 2022 IBM
+#####################################################
+
+module "volume" {
```

Comment:
Add detect secret hook in pre-commit file (Take code from https://github.com/terraform-ibm-modules/terraform-ibm-module-template repo)
https://github.com/VaishnaviGopal/terraform-ibm-vpc-vsi/blob/main/.pre-commit-config.yaml

Reply:
done